.travis.yml

```diff
@@ -17,3 +17,4 @@ script:
   - cd /home/travis/build/Chris-Johnston/Easier68k
   - (find easier68k -iname "*.py" | xargs pylint) || true
   - (find tests/ -iname "*.py" | xargs pylint) || true
+  - py.test --cov easier68k tests/

```

Comment:
Is this changing the working directory? Because that might be why it cannot find the `easier68k/../../*.x68` file. Have you tried using either `py.test --cov easier68k $(pwd)` or `py.test --cov easier68k .`?

Reply:
I have tried both suggestions, and it still cannot find those files. What's most interesting about it is that both of those files are in the same directory as the tests that call upon them. Could there be something wrong with the tests? They seem to be passing when you run pytest, so I'm not sure that's it either.

One thing we could try is changing the file paths to be relative for both of the affected tests.